REQUIRE

```diff
@@ -1,3 +1,4 @@
 julia 0.7
 MathProgBase 0.7 0.8
 DataStructures
+Requires
```

Comment:
I'll note that Requires tends to significantly slow down package load times (I looking into it for Memento and FilePaths) and I'm not sure how well it'll play with system images.

Reply:
Requires is supposed to speed up load times, that was it's original purpose.

Reply:
"supposed to"

Reply:
https://github.com/MikeInnes/Requires.jl/issues/39